### ISSUE TITLE
First EGO example not working.

### ISSUE THREAD

**Issue by TOP1RM** [SMTorg/smt#362]
Hello,

I am Rémi MOCHON, a PhD students who intend to use some constant optimization of a control law. I have already used EGO with Uranie (based on root, developped by the CEA), but I would like to test smt. I managed to install it but the first example for EGO (_https://smt.readthedocs.io/en/latest/_src_docs/applications/ego.html_) does not work. Python throws this error (along with a lot of warnings about true-divide I read it was from dead code not affecting the results) :

```
Traceback (most recent call last):
  File "home/?????/ego_tests.py", line 39, in <module>
    y_ei_plot = -ego.EI(x_plot, y_data_k)
  File "/home/?????/lib/python3.9/site-packages/smt/applications/ego.py", line 221, in EI
    if enable_tunneling:
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

So I dived into the code and found that the EI function is defined as follow:
```
def EI(self, points, enable_tunneling=False, x_data=None):
```
And the line in the example that make this error appear is:
```
y_ei_plot=-ego.EI(x_plot, y_data_k)
```
So it explains why the error is thrown. By removing the second argument `y_data_k`, the code works (I have the nice figures, exactly as in the website), but I am not sure what I am doing since the function is not documented. Could you tell me if I am doing non sense by removing the second argument, or not ? Or provide some documentation to make me understand what I am doing. I just intend to use EGO.

Looking forward for your answer.

Best regards,
Rémi MOCHON

**Comment by Paul-Saves** [comment]
Thank you Rémi Mochon (@TOP1RM ) to notice us ! 

@relf It has been changed by @Laurentww  in #346  (https://github.com/SMTorg/smt/pull/346/files).

 EI(self, points, y_data, enable_tunneling=False, x_data=None) was changed to 
 EI(self, points, enable_tunneling=False, x_data=None). 

The y_data options was deleted. 

However, the documentation and tests were not updated !

**Comment by relf** [comment]
Thanks for the heads up! Actually EGO code examples were already updated [here](https://github.com/SMTorg/smt/blob/d90aba9e0a77f1c4dcb2c7aaef80903076ee9b06/smt/applications/tests/test_ego.py#L554) as they are tested but it seems I forgot to  commit the corresponding generated doc file. This is fixed now by d90aba9.

**Comment by Paul-Saves** [comment]
Perfect, thank you Rémi ! :)
